<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAMainWindow</class>
 <widget class="QMainWindow" name="SAMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ship Agency Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>991</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="sendApplication">
     <attribute name="title">
      <string>Application</string>
     </attribute>
     <widget class="QStackedWidget" name="procedure">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>481</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="works">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>851</width>
          <height>441</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="workInfo">
           <column>
            <property name="text">
             <string>Work Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Work Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Standard</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Abbreviation</string>
            </property>
            <property name="font">
             <font>
              <underline>false</underline>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Code</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tugboat needed</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Select</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="lastSection0">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="confirmWorks">
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>0</y>
          <width>701</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Choose Works : select the work and enter its corresponding number of Tugboats you need</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="port">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>70</y>
          <width>321</width>
          <height>391</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="2">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="selectPort"/>
           </item>
           <item row="4" column="2">
            <widget class="QDateTimeEdit" name="startTime"/>
           </item>
           <item row="1" column="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="2">
            <widget class="QDateTimeEdit" name="endTime"/>
           </item>
           <item row="3" column="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>End Time</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Phone</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Start Time</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QLineEdit" name="shipTel"/>
           </item>
           <item row="9" column="2">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="2">
            <widget class="QComboBox" name="business">
             <item>
              <property name="text">
               <string>内贸</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>外贸</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="10" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="lastSection1">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="submitApplication">
               <property name="text">
                <string>Submit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>20</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Other Details</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ship">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>941</width>
          <height>461</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Check vessel information:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="zhName"/>
           </item>
           <item>
            <widget class="QPushButton" name="findShip">
             <property name="text">
              <string>Check</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="shipInfo"/>
         </item>
         <item>
          <widget class="QPushButton" name="confirmShip">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Table</string>
     </attribute>
     <widget class="QTableWidget" name="ordersTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>971</width>
        <height>401</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Application ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Boat Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Submission Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Billing Clerk</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Audit Clerk</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>430</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="lastPage">
         <property name="text">
          <string>Previous Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentPage">
         <property name="text">
          <string>当前页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextPage">
         <property name="text">
          <string>Next Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalPage">
         <property name="text">
          <string>总页数</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="logOut">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>420</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
